app/code/Magento/Ui/view/base/web/js/modal/modal.js

```diff
@@ -104,7 +104,6 @@ define([
                 /**
                  * Escape key press handler,
                  * close modal window
-                 * @param {Object} event - event
```

Comment:
Why you remove param description if this param still passing to `escapeKey` method?

Reply:
Sorry, I have missed the parameter description but now I have added